orchestration/src/main/java/com/sap/ai/sdk/orchestration/ImageItem.java

```diff
@@ -0,0 +1,56 @@
+package com.sap.ai.sdk.orchestration;
+
+import java.util.Locale;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Represents an image item in a {@link MessageContent} object.
+ *
+ * @param imageUrl the URL of the image
+ * @param detailLevel the detail level of the image (optional
+ */
+public record ImageItem(@Nonnull String imageUrl, @Nonnull DetailLevel detailLevel)
+    implements ContentItem {
+
+  /**
+   * Creates a new image item with the given image URL and detail level.
+   *
+   * @param imageUrl the URL of the image
+   * @param detailLevel the detail level of the image
+   */
+  public ImageItem(@Nonnull final String imageUrl, @Nullable final DetailLevel detailLevel) {
+    this.imageUrl = imageUrl;
+    this.detailLevel = detailLevel != null ? detailLevel : DetailLevel.auto;
+  }
+
+  /**
+   * Creates a new image item with the given image URL.
+   *
+   * @param imageUrl the URL of the image
+   */
+  public ImageItem(@Nonnull final String imageUrl) {
+    this(imageUrl, DetailLevel.auto);
+  }
```

Comment:
These two seem redundant, why would we need `new ImageItem(url, null)` if we have `new ImageItem(url)`?  (also applies to `MessageContent` & `UserMessage`)